core/src/main/java/dev/hypera/scaffolding/schematic/readers/SpongeSchematicReader.java

```diff
@@ -24,21 +24,17 @@
 
 import dev.hypera.scaffolding.schematic.NBTSchematicReader;
 import dev.hypera.scaffolding.schematic.Schematic;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionException;
 import net.minestom.server.instance.block.Block;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 import org.jglrxavpok.hephaistos.collections.ImmutableByteArray;
 import org.jglrxavpok.hephaistos.nbt.NBTCompound;
 import org.jglrxavpok.hephaistos.nbt.NBTException;
 
+import java.util.*;
```

Comment:
`*` imports should not be used, `java.` imports should not be placed separately from other imports.

Reply:
I cant reverse those, the imports are made not by myself, i dont wanna figure out which exact imports i used. If you have an idea on how to change settings in intelliJ for those imports, i would be happy to hear them.